### ISSUE TITLE
Question about science data

### ISSUE THREAD

**Issue by Its-Just-Luci** [Kerbalism/Kerbalism#571]
Is there a way to view the amount of science you will receive once transmission is complete, and is there any way to increase transmission speed

**Comment by gotmachine** [comment]
You can increase transmission speed by using a more powerful antenna. Using multiple antennas won't do anything to the bandwidth (it only increase range).

As for the science value of things, there is a popup "info" window that you can open from the part action menu or from the Kerbalism automation tab. You can also see the science value of files currently stored when you hover on them in the file manager.
